### PR TITLE
Sam port not test

### PR DESCRIPTION
**Pull request by Sameeranjoshi**

### CODE REVIEW

**Comment by schweitzpgi** [comment]
are you seeing this?

```
FAIL: Flang :: Fir/character.fir (10 of 459)
******************** TEST 'Flang :: Fir/character.fir' FAILED ********************
Script:
--
: 'RUN: at line 1';   tco -emit-fir /work/flang/test/Fir/character.fir | tco | FileCheck /work/flang/test/Fir/character.fir
--
Exit Code: 2
```

**Comment by Sameeranjoshi** [comment]
Ok, looks like the commit message didn't come on top on PR it GitHub UI.

Yes, it's failing as I couldn't figure the test to be correct.

File: flang/test/Fir/character.fir [hunk 1]
```diff
@@ -1,12 +1,17 @@
+// RUN: tco -emit-fir %s | tco | FileCheck %s
+// UNSUPPORTED: !fir
```
Comment (schweitzpgi) [inline flang/test/Fir/character.fir:2]: Why this?

File: flang/test/Fir/constant.fir [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+  // TODO: What's that number?
+  return %0 : !fir.real<10>
+}
+
+// CHECK-LABEL: define i16 @z()
+func @z() -> !fir.logical<2> {
+  %1 = constant true
+  %0 = fir.convert %1 : (i1) -> !fir.logical<2>
+ // CHECK-LABEL: ret i16 -1
+ // TODO: Why is it -1?
```
Comment (schweitzpgi) [inline flang/test/Fir/constant.fir:24]: It's the value `1 : i1` sign extended to an `i16`.

**Pull request merged**